src/main/asciidoc/_chapters/hbase-default.adoc

```diff
@@ -763,9 +763,10 @@ Maximum retries. This is a maximum number of iterations
 Request Compaction after bulkload immediately.
 If bulkload is continuous, the triggered compactions may increase load,
 bring about performance side effect.
+(During HBase-2.0, "hbase.compaction.after.bulkload.enable" default value is true)
```

Comment:
'For HBase-2.x, the default value for "hbase.compaction.after.bulkload.enable" is true'.

Reply:
yes, I amend it